### DISCUSSION TITLE
Table component for Show operations

### DISCUSSION THREAD

**Discussion by anxgstadler** [Laravel-Backpack/community-forum#1188] [Q&A (Help)] [answered]
I'm new to Laravel Backpack and evaluating my options...

I'm looking for a table component to list (and edit/create) related models within the show operation of an entity.

I found a few answers on stackoverflow that are 2 years old or older saying that such a component was already on its way - but I still can't find it, not even in the documentation of 7.0-dev. (And the github repositories from 3rd parties that were linked don't seem to be actively maintained anymore or don't provide any documentation at all.)

The only one I found that looks somehow recent (if you want to count having the last update over a year ago "recent") is https://github.com/izica/relations-widgets-for-backpack

How is the current status of this issue, or is there already a solution to this problem?

I'm evaluating Backback currently for a new project, and this is a requirement I will have to deal with a lot.

Thanks! :-)

**Comment by siberfx** [comment]
I can advise you to create your own show blades and build it the way you desire to, even you can use livewire, vue, react typescript etc. whatever technology stack you desire to use. inside the crud controller you can just show the CRUD::showView('new-custom.blade-file-path')  
you dont have to stick with whats been shipped with laravel only, its compeltely flexible.
Cheers

**Comment by karandatwani92** [comment, marked as answer]
Hey @anxgstadler 

It's not a built-in feature yet. But there are many ways to achieve this. You can choose according to your preference and requirements.

1. Izica package: https://github.com/izica/relations-widgets-for-backpack
2. Create a custom [`widget`](https://backpackforlaravel.com/docs/6.x/base-widgets#creating-a-custom-widget-type-1) or [`column`](https://backpackforlaravel.com/docs/6.x/crud-columns#creating-a-custom-column-type-1) only **or** a complete `show` page.
3. Or link it using a subfields:
```
CRUD::column([
    'name' => 'activities',
    'type' => 'relationship',
    'label' => 'Logs',
    'subfields' => [
        ['name' => 'log_name'],
        ['name' => 'description', 'limit' => '100'],
        ['name' => 'created_at', 'label' => 'Created At(CHECK SYS)'],
        [
            'name'     => 'link',
            'label'    => 'Custom HTML',
            'type'     => 'custom_html',
            'value'    => fn($entry) => '<a href="/entity/' . $entry->id . '/show" class="btn-primary" target="_blank">View</a>',
        ],
    ]
]);
```

**Reply by anxgstadler** [reply]
Thanks! :-)
